src/DependencyInjection/Compiler/ThumbnailCompilerPass.php

```diff
@@ -32,13 +32,18 @@ public function process(ContainerBuilder $container)
             'sonata.media.thumbnail.format'
         );
 
+        $addResizerMethod = 'addResizer';
```

Comment:
Is there a reason why you use a variable? Can we use a constant here?

Reply:
No other reason than to avoid the potential mistake to check a different method than the one added later. Should I just remove this variable?